<launch>
  <node pkg="path_sim" type="c3" name="c3" output="screen" >
  <param name="raw_topic" value="/mobile_base/commands/velocity/t" />
  <param name="vel_topic" value="/mobile_base/commands/velocity" />
  <param name="frequency" value="200" />
  <param name="v_max_vel" value="0.3" />
  <param name="v_min_vel" value="-0.05" />
  <param name="v_max_acc" value="2" />
  <param name="v_min_acc" value="-2" />
  <param name="v_jeck" value="2.0" />
  <param name="a_max_vel" value="0.5" />
  <param name="a_min_vel" value="-0.5" />
  <param name="a_max_acc" value="5" />
  <param name="a_min_acc" value="-5" />
  <param name="a_jeck" value="5.0" />
  </node>
</launch>
